src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -97,19 +97,23 @@ MR <: MempoolReader[TX]](networkControllerRef: ActorRef,
     networkControllerRef ! SendToNetwork(msg, Broadcast)
   }
 
-  protected def viewHolderEvents: Receive = {
+  private def transactionEvents: Receive = {
```

Comment:
This change seems ok to me. But if we intend to allow users of Scorex to create blockchain systems that subclass `NodeViewSynchronizer` and override its `Receive` methods, then we should keep this `protected` instead of `private`.

@kushti, what do you think?

Reply:
I think that this one for sure should be protected